snapcore/snapweb
# Visual design tweaks based on design sweep

## Done

This PR contains design tweaks based on a review carried out by myself and Karl Waghorn-Moyce.

The full list of bugs/tweaks can be found here: https://github.com/CanonicalLtd/snapweb-design/issues?q=is%3Aissue+is%3Aopen+label%3A%22fix+comitted%22

There should be no functional alterations contained in this changeset.

Each commit relates to a single design tweak and most commits have a reference to the issue in the commit log.

## QA

- Check code to ensure nothing looks out of place. 
- Run project and click through each template checking that none of the pages are broken.
- When this code has been merged and eployed to the demo site, Karl will run through and check off each design tweak now looks as expected.
